### DISCUSSION TITLE
Trigger post-processing with bash command?

### DISCUSSION THREAD

**Discussion by josh-aliencode** [pymedusa/Medusa#10755] [Q&A]
Is there a way to trigger the post processing in pymedusa with a command?
It would be handy for me to use in a script.

Something similar worked with Sickrage:
`curl -s "http://<HOST:PORT/WEBROOT>/api/v1/<API KEY>/?cmd=postprocess&path=<DOWNLOAD DIR>"`

**Comment by p0psicles** [comment]
It has been asked / discussed before. But you'll have to use the api/v2 command. There is a version of it that runs sync and one that runs async. I would recommend to use the async command.

You could see the command by analyzing the browsers requests F12 (network tab). When you start a manual pp task in the ui

**Reply by josh-aliencode** [reply]
I appreciate the nudge here, I can see the post information under F12 now..
But since I'm new to this and the api.. how do i verify credentials with the request? I can find my api key.. but i'm not sure where to use it in the curl command.

This is as far as i've gotten:
`curl -k -X POST -d "proc_dir=/incoming/Medusa&process_method=move&force=true&is_priority=true&delete_on=false&failed=false&proc_type=manual&ignore_subs=false&run_async=true" https://127.0.0.1:8081/home/postprocess/processEpisode`

**Reply by josh-aliencode** [reply]
Edit.. Nevermind.. I didn't realize I could literally copy the command under the F12. Thanks so much!
